lib/Widget/dashboard/pre_operation/preDashboardDetail.dart

```diff
@@ -61,7 +61,7 @@ class _PreDashboardDetailState extends State<PreDashboardDetail> {
                               child: AbsorbPointer(
                                 absorbing: !_evaluationFormViewModel
                                     .disableVirtalSignButton2(false),
```

Comment:
change this to vital sign

Reply:
also every disableVirtalSignButton , please recheck again na

Reply:
Solve in 98106d0 ka